src/test/resources/SimpleTestClass.java

```diff
@@ -0,0 +1,19 @@
+package org.example.javaparser.samples.Test;
+
+import java.util.Random;
+import java.util.stream.IntStream;
+
+/**
+ * This is a testing class.
+ */
+public class SimpleTestClass { // test class
+    public static void main(String[] args) {
+        var k = (new Random()).nextInt(1500);
+        IntStream.range(0, 1000).forEach( i -> {
+            k++;
+            if (!k%5) {
+                System.out.println(k + "5 is 0!");
+            }
+        });
+    }
+}
```

Comment:
```suggestion
```

Reply:
I use this Java class to test the tokenization process. Do you prefer I store this file in a different GH repo and then use the GHProvider to get it?

Reply:
Oh, I see. It's fine as it is, then.